File: src/main/java/br/com/dusty/dkits/gamer/PrimitiveGamer.java
package br.com.dusty.dkits.gamer;

import java.util.UUID;

public class PrimitiveGamer {
	
	private String uuid;
	
	private int id = -1;
	
	int kills = 0, deaths = 0, killStreak = 0, maxKillStreak = 0;
	float xp = 0, money = 0;
	int hgWins = 0, hgLoses = 0;
	
	UUID getUUID() {
		return UUID.fromString(uuid);
	}
	
	void setUUID(UUID uuid) {
		this.uuid = uuid.toString();
	}
	
	@Override
	public String toString() {
		return "PrimitiveGamer{" + "uuid='" + uuid + '\'' + ", id=" + id + ", kills=" + kills + ", deaths=" + deaths + ", killStreak=" + killStreak + ", maxKillStreak=" + maxKillStreak + ", xp=" + xp + ", money=" + money + ", hgWins=" + hgWins + ", hgLoses=" + hgLoses + '}';
	}
}

File: src/main/kotlin/br/com/dusty/dkits/gamer/PrimitiveGamer.kt
package br.com.dusty.dkits.gamer

import java.util.*

class PrimitiveGamer(var uuid: String) {

	var id = -1

	var kills = 0
	var deaths = 0
	var killStreak = 0
	var maxKillStreak = 0
	var xp = 0.0
	var money = 0.0
	var hgWins = 0
	var hgLosses = 0
	var oneVsOneWins = 0
	var oneVsOneLosses = 0

	fun getUniqueId(): UUID = UUID.fromString(uuid)

	fun setUniqueId(uniqueId: UUID): PrimitiveGamer {
		this.uuid = uniqueId.toString()

		return this
	}
}
